weatherAplication/Modules/Home/View/SearchButton.swift

```diff
@@ -0,0 +1,19 @@
+import Foundation
+import UIKit
+
+class SearchButton: UIButton {
```

Comment:
to powinno nie powinno znajdować się w folderze Home tylko w folderze na customowe widoczki, typu customView